.github/workflows/testing.yml

```diff
@@ -32,6 +32,20 @@ jobs:
       run: |
         sudo apt-get install virtualenv libkrb5-dev
         pip install -r devel.txt
+    
+    - name: Prepare for integration test
+      if: matrix.make-command == 'integration_test'
+      run: |
+          echo "[tcms]" > ~/.tcms.conf
+          echo "url = https://tcms.kiwitcms.org/xml-rpc/" >> ~/.tcms.conf
+          echo "username = kiwitcms-bot" >> ~/.tcms.conf
+          echo "password = ${{ secrets.TCMS_PASSWORD }}" >> ~/.tcms.conf
+          export TCMS_PRODUCT=$GITHUB_REPOSITORY
+          # branch name or pull/123
+          export TCMS_PRODUCT_VERSION=$(echo $GITHUB_REF | sed "s|refs/heads/||" | sed "s|refs/||" | sed "s|/merge||")
+          # short commit number
+          export TCMS_BUILD=$(echo $GITHUB_SHA | cut -c1-7)
+
```

Comment:
For the SSL verification failure see https://github.com/kiwitcms/junit.xml-plugin/blob/master/.travis.yml#L18.